src/components/Admin/ListSkills/ListSkills.js

```diff
@@ -493,7 +490,7 @@ class ListSkills extends React.Component {
                     record.language,
                     record.reviewStatus,
                     record.editStatus,
-                    record.skillTag,
+                    record.skill_tag,
```

Comment:
@Akshat-Jain Please change to camel case, as pointed out in https://github.com/fossasia/susi_skill_cms/pull/1388

Reply:
Done. Please review. Thanks :-)